src/persist-client/src/error.rs

```diff
@@ -23,6 +23,14 @@ impl std::fmt::Display for InvalidUsage {
 
 impl std::error::Error for InvalidUsage {}
 
+#[cfg(test)]
+impl PartialEq for InvalidUsage {
+    fn eq(&self, other: &Self) -> bool {
+        // TODO: What are the right semantics here?
```

Comment:
Seems fine for `cfg(test)`

Reply:
yeah, I think I agree. done. also removed the one this was copied from (ExternalError)